spieleplattform-backend/src/test/kotlin/com/example/spieleplattformbackend/game/GameServiceTest.kt

```diff
@@ -53,7 +53,7 @@ class GameServiceTest {
     }
 
     @Test
-    fun whenGetGamesByGameConsoleId_thenReturnGames() {
+    fun `when getGamesByGameConsoleId with id=1 return games`() {
```

Comment:
Das ist zwar ein bisschen aussagekräftiger als vorher, aber sagt nicht wirklich aus was du hier testest. Du willst ja nicht sicherstellen, dass du immer ein game zurückbekommst wenn die id 1 ist und erst recht nicht, dass du niemals ein Game zurückbekommst wenn die id 2 ist. Was du hier testest ist eher `when consolId exists then getGamesByConsoleId returns` und `when consolId does not exist then getGamesByConsolId returns empty list` oder so ähnlich.

Reply:
Bis auf das Naming aber gute Testfälle :+1: